src/features/surveys/components/SurveyEditor/blocks/ChoiceQuestionBlock.tsx

```diff
@@ -0,0 +1,13 @@
+import { Box } from '@mui/material';
+import { FC } from 'react';
+import { ZetkinOptionsQuestion } from 'utils/types/zetkin';
+
+interface ChoiceQuestionBlockProps {
+  question: ZetkinOptionsQuestion;
+}
+
+const ChoiceQuestionBlock: FC<ChoiceQuestionBlockProps> = ({ question }) => {
+  return <Box>{JSON.stringify(question)}</Box>;
```

Comment:
How do you think when you decide to use FC or "just" a `const Component = () => {/...}` type syntax (where i guess JSX.Element is inferred as return type)? I wonder att times and I feel like I don't have the hang of this.